src/index.ts

```diff
@@ -228,7 +228,7 @@ export class EsbuildPlugin implements Plugin {
       this.rootFileNames.map(async ({ entry, func, functionAlias }) => {
         const config: Omit<BuildOptions, 'watch'> = {
           ...this.buildOptions,
-          external: [...this.buildOptions.external, ...this.buildOptions.exclude],
+          external: [...this.buildOptions.external, ...removeWildCardStrings(this.buildOptions.exclude)],
```

Comment:
Since the plugin passes the packages included in `exclude` currently, I tried not to disturb that behavior. But the wild card/patterns don't make any sense to pass, so removing them here, including only the plain strings.